content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+The authors use Blosum encoding for 9-mer peptides.
+Vang et al. [@doi:10.1093/bioinformatics/btx264] combined deep learning with tools from natural language processing to approach 9-mer peptides.
+They employ a technique called distributed representation to map amino acids onto a 15-dimensional vector space based on their context relation to other amino acids.
+A convolutional neural network, titled HLA-CNN, takes peptide sequences and replaces individual amino acids with their corresponding vectors to predict binding.
+
+Deep learning's unique flexibility was recently leveraged by Bhattacharya et al., who used a gated recurrent neural network method called MHCnuggets to overcome the difficulty of multiple length peptides.
+Under this framework, they used smoothed sparse encoding to represent amino acids individually.
+Because MHCnuggets had to be trained for every MHC allele, performance was far better for alleles with abundant, balanced training data.
+
+In a comparison of several current methods, Bhattacharya et al. found that the top methods — NetMHC, NetMHCpan, MHCflurry, and MHCnuggets — showed comparable performance, but large differences in speed.
+Convolutional neural networks (in this case, HLA-CNN) showed comparatively poor performance, while shallow and recurrent neural networks performed the best.
+They found that MHCnuggets — the recurrent neural network — was by far the fastest training among the top performing methods.
+In predicting interactions between proteins, deep learning has achieved state-of-the-art results and shows promise to overcome previous challenges in the field.
+
+### PPI networks and graph analysis
+
+The abundance of large networks of biological data presents a different approach to computational prediction.
+Because interacting proteins are more likely to share a similar function, the connectivity of a PPI network itself can be a valuable information source for the prediction of function [@doi:10.1038/msb4100129].
+To incorporate higher-order network information, it is necessary to find a lower-level embedding of network structure that preserves this higher-order structure.
+Rather than use hand-crafted network features, deep learning shows promise for the automatic discovery of predictive features within networks.
+For example, Navlakha [@doi:10.1162/NECO_a_00294] showed that a deep autoencoder was able to compress a graph to 40% of its original size, while being able to reconstruct 93% of the original graph's edges.
+The deep learning method had greater reconstruction accuracy than comparable dimensionality-reduction methods such as PCA with SVD.
+While the activity of some hidden layers correlated with several popular hand-crafted network features, this work showed that deep learning can effectively reduce graph dimensionality while retaining much of its structural information.
```

Comment:
Perhaps name one or two of the standard features: `hand-crafted network features such as... , this work`.